src/parameterized/LKJL.jl

```diff
@@ -24,8 +24,20 @@ struct LKJL{k, N, T} <: ParameterizedMeasure{N}
     par :: NamedTuple{N,T}
 end
 
+LKJL{k}(nt::NamedTuple{N,T}) where {k,N,T} = LKJL{k,N,T}(nt)
+
 LKJL(k,η) = LKJL{k, (:η,), Tuple{typeof(η)}}((η=η,))
 
+LKJL(k) = LKJL(k, 1.0)
```

Comment:
Perhaps it's better to restrict the type of `k` so a user is less like to put eta here, and use integer `1` for eta to avoid promoting to `Float64` in the log density if the user passes a matrix with an eltype like `Float32`:
```suggestion
LKJL(k::Integer) = LKJL(k, 1)
```